### PR TITLE
Cleaned Up Web Browser Commands

### PR DESCRIPTION
**Pull request by openbots-ff**

### CODE REVIEW

File: taskt/Core/Automation/Commands/Web Browser Commands/SeleniumCreateBrowserCommand.cs [hunk 1]
```diff
@@ -1,112 +1,118 @@
...
+        [XmlAttribute]
+        [PropertyDescription("Instance Tracking")]
+        [PropertyUISelectionOption("Forget Instance")]
+        [PropertyUISelectionOption("Keep Instance Alive")]
+        [InputSpecification("Select **Forget Instance** to forget the instance after execution finishes, " +
+                            "or **Keep Instance Alive** to allow subsequent tasks to call the instance by name.")]
+        [SampleUsage("")]
+        [Remarks("Calling the **Close Browser** command or ending the browser session will end the instance. " +
+                 "This command only works during the lifetime of the application. " +
+                 "If the application is closed, the references will be forgetten automatically.")]
```
Comment (ob-ab) [inline taskt/Core/Automation/Commands/Web Browser Commands/SeleniumCreateBrowserCommand.cs]: Correct Spellings.
```suggestion
                 "If the application is closed, the references will be forgotten automatically.")]
```

File: taskt/Core/Automation/Commands/Web Browser Commands/SeleniumGetBrowserInfoCommand.cs [hunk 2]
```diff
@@ -84,31 +85,24 @@ public override void RunCommand(object sender)
                 default:
```
Comment (ob-ab) [inline taskt/Core/Automation/Commands/Web Browser Commands/SeleniumGetBrowserInfoCommand.cs:86]: Clean up namespace `Newtonsoft.Json` on line 83.

File: taskt/Core/Automation/Commands/Web Browser Commands/SeleniumElementActionCommand.cs [hunk 3]
```diff
@@ -787,64 +712,61 @@ public void seleniumAction_SelectionChangeCommitted(object sender, EventArgs e)
...
                 default:
                     break;
             }
-
-            ElementsGridViewHelper.DataSource = v_WebActionParameterTable;
+            _elementsGridViewHelper.DataSource = v_WebActionParameterTable;
         }
 
-        private void ElementsGridViewHelper_CellContentClick(object sender, DataGridViewCellEventArgs e)
+        private void _elementsGridViewHelper_CellContentClick(object sender, DataGridViewCellEventArgs e)
```
Comment (ob-ab) [inline taskt/Core/Automation/Commands/Web Browser Commands/SeleniumElementActionCommand.cs]: PascalCase naming convention for Event Handlers.

File: taskt/Core/Automation/Commands/Web Browser Commands/SeleniumElementActionCommand.cs [hunk 4]
```diff
@@ -666,36 +608,27 @@ public void ShowRecorder(object sender, EventArgs e)
 
         public void seleniumAction_SelectionChangeCommitted(object sender, EventArgs e)
```
Comment (ob-ab) [inline taskt/Core/Automation/Commands/Web Browser Commands/SeleniumElementActionCommand.cs]: Please follow the PascalCase Naming convention for all the event handlers and also update their references.

File: taskt/Core/Automation/Commands/Web Browser Commands/SeleniumElementActionCommand.cs [hunk 5]
```diff
@@ -305,7 +280,6 @@ public override void RunCommand(object sender)
                                             where rw.Field<string>("Parameter Name") == "Attribute Name"
                                             select rw.Field<string>("Parameter Value")).FirstOrDefault().ConvertToUserVariable(sender);
 
-
                     var optionsItems = new List<string>();
                     var ele = (IWebElement)element;
                     var select = new SelectElement(ele);
```
Comment (ob-ab) [inline taskt/Core/Automation/Commands/Web Browser Commands/SeleniumElementActionCommand.cs:286]: Please clean up **Script** namespace reference on line 298 and 383.

**Review by ob-ab — CHANGES_REQUESTED** [review]
Please make a few small changes.

File: taskt/Core/Automation/Commands/Web Browser Commands/SeleniumGetBrowserInfoCommand.cs (hunk 2 above)
Comment (openbots-ff) [reply taskt/Core/Automation/Commands/Web Browser Commands/SeleniumGetBrowserInfoCommand.cs:86]: done

File: taskt/Core/Automation/Commands/Web Browser Commands/SeleniumElementActionCommand.cs (hunk 3 above)
Comment (openbots-ff) [reply taskt/Core/Automation/Commands/Web Browser Commands/SeleniumElementActionCommand.cs]: done

File: taskt/Core/Automation/Commands/Web Browser Commands/SeleniumElementActionCommand.cs (hunk 4 above)
Comment (openbots-ff) [reply taskt/Core/Automation/Commands/Web Browser Commands/SeleniumElementActionCommand.cs]: done

File: taskt/Core/Automation/Commands/Web Browser Commands/SeleniumElementActionCommand.cs (hunk 5 above)
Comment (openbots-ff) [reply taskt/Core/Automation/Commands/Web Browser Commands/SeleniumElementActionCommand.cs:286]: done

**Review by ob-ab — APPROVED** [review]

**Review by CameronHerwig — APPROVED** [review]

**Pull request merged**